cmake/find_ltdl.cmake

```diff
@@ -1,28 +1,30 @@
 
 # /usr/bin/x86_64-linux-gnu-ld: /usr/lib/gcc/x86_64-linux-gnu/8/../../../x86_64-linux-gnu/libltdl.a(libltdl_libltdl_la-lt__alloc.o): relocation R_X86_64_PC32 against symbol `stderr@@GLIBC_2.2.5' can not be used when making a shared object; recompile with -fPIC
 if (NOT BUILD_SHARED AND "${CMAKE_SYSTEM_NAME}" STREQUAL "Linux")
+    # using `uname -v` rather that `lsb_release -is` since some Ubuntu or Debian-based disros (hello KDE Neon)
+    # have different distribution ID, but retain Ubuntu or Debian in kernel version.
     execute_process(
-        COMMAND lsb_release -is
-        OUTPUT_VARIABLE _distname
+        COMMAND uname -v
+        OUTPUT_VARIABLE _dist_info
         OUTPUT_STRIP_TRAILING_WHITESPACE
     )
-    if (_distname)
+    if (_dist_info)
         message (STATUS "lsb_release -is: ${_distname}")
```

Comment:
`_distname` => `_dist_info` inside the message? aslo message itself updated?

Reply:
removed that part